spec.html

```diff
@@ -29029,7 +29029,7 @@ <h1>
               1. Let _Octets_ be the List of octets resulting by applying the UTF-8 transformation to _cp_.[[CodePoint]].
               1. For each element _octet_ of _Octets_, do
                 1. Let _hex_ be the String representation of _octet_, formatted as an uppercase hexadecimal number.
-                1. Set _R_ to the string-concatenation of _R_, *"%"*, and ! StringPad(_hex_, *2*<sub>𝔽</sub>, *"0"*, ~start~).
+                1. Set _R_ to the string-concatenation of _R_, *"%"*, and StringPad(_hex_, *2*<sub>𝔽</sub>, *"0"*, ~start~).
```

Comment:
The 𝔽 indicates a `Number` value, but we need to pass a mathematical integer.

```suggestion
                1. Set _R_ to the string-concatenation of _R_, *"%"*, and StringPad(_hex_, 2, *"0"*, ~start~).
```